python/tvm/relay/op/contrib/dnnl.py

```diff
@@ -34,10 +34,13 @@
 """
 import logging
 
-import tvm.ir
+import tvm
```

Comment:
Usually we won't import whole TVM. Can you only import the used part?

Reply:
Fixed. Removed `import tvm`.